lighthouse-cli/index.js

```diff
@@ -29,6 +29,7 @@ const yargs = require('yargs');
 const Printer = require('./printer');
 const lighthouse = require('../lighthouse-core');
 const assetSaver = require('../lighthouse-core/lib/asset-saver.js');
+const Launcher = require('./chrome-launcher');
```

Comment:
maybe call something like `ChromeLauncher` so it's clear that it's responsible for launching Chrome and not Lighthouse?